include/vcpkg/build.h

```diff
@@ -221,6 +221,7 @@ namespace vcpkg::Build
 
         Triplet triplet;
         bool load_vcvars_env = false;
+        bool no_compiler_tracking = false;
```

Comment:
I think the code is easier to read if you change the variable to `compiler_tracking`. Then you don't have double negations like `if (... && !abi_info.pre_build_info->no_compiler_tracking)`

Reply:
Yeah, I had the same concern... My rationale was that as of now, the default is that there is compiler tracking. Hence, you need to add an option to _prevent_ compiler tracking. This is why I named the option `VCPKG_NO_COMPILER_TRACKING`. From this point, my concern was to be consistent, so that if somebody want's to see how this functionality is implemented, they can use something like `grep --ignore-case VCPKG_NO_COMPILER_TRACKING` and `grep --ignore-case NO_COMPILER_TRACKING`.

I'd like to have some more feedback on that point. I'm happy to change either the option name `VCPKG_NO_COMPILER_TRACKING` or even to break consistency if the majority of people prefer this option.

Reply:
I do like the `NO_COMPILER_TRACKING`, rather than `COMPILER_TRACKING`. However, I think we might want to rename it to `DISABLE_COMPILER_TRACKING`: `VCPKG_DISABLE_COMPILER_TRACKING` and `disable_compiler_tracking`. @ras0219, what are your thoughts?

Reply:
`VCPKG_DISABLE_COMPILER_TRACKING` is clear IMO.

Reply:
Great I'll use `VCPKG_DISABLE_COMPILER_TRACKING`!